artemis-server/src/main/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImpl.java

```diff
@@ -1513,8 +1513,7 @@ public ConfigurationImpl setResolveProtocols(boolean resolveProtocols) {
          TransportConfiguration connector = getConnectorConfigurations().get(connectorName);
 
          if (connector == null) {
-            ActiveMQServerLogger.LOGGER.warn("bridgeNoConnector(connectorName)");
-
+            ActiveMQServerLogger.LOGGER.connectionConfigurationIsNull();
```

Comment:
Need to supply the connector name in the log output, so if connector is not found, you know what is missing.